holder/wallet/index.js

```diff
@@ -277,11 +280,10 @@ try {
     })
     .then(() => {
       console.log('Done!');
-      process.exit(0);
     })
     .catch((err) => {
-      console.error(err);
-    });
+      console.error(err.message);
+    }).finally(() => process.exit(0));
```

Comment:
Should use a non-zero exit code in error case.

Reply:
this is just there because node.js thread doesn't exist.